.github/workflows/main.yml

```diff
@@ -29,7 +29,25 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: '10.x'
-         
+
+      - name: Npm cache Posix
+        if: startsWith(matrix.os, 'ubuntu') || startsWith(matrix.os, 'macOS')
+        uses: actions/cache@v1
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
```

Comment:
Maybe you can use `unix-node-${{ hashFiles('**/package-lock.json') }}`

Reply:
To share between MAC and Ubuntu

Reply:
Yeah makes sense